exaparser/workflow/units/execution/scripting/shell_with_results.py

```diff
@@ -32,6 +34,12 @@ def _is_espresso_calculation(self):
         if find_file_with_pattern(ESPRESSO_INPUT_FILE_REGEX, self.work_dir):
             return True
 
+    def _has_aiida_archive_zip_file(self):
+        zip_file = find_file('.zip', self.work_dir)
```

Comment:
Same as in https://github.com/Exabyte-io/express/pull/56/files#r544599480

Reply:
Looks like the link is no longer available, do you remember what you were referring to?